website/src/content/api-reference/commercetools-frontend-application-shell-connectors.mdx

```diff
@@ -0,0 +1,319 @@
+---
+title: "@/application-shell-connectors"
+---
+
+<Subtitle>
+
+Components to work with the `@commercetools-frontend/application-shell` package to access and subscribe to specific data within the AppShell.
+
+</Subtitle>
+
+# Installation
+
+```
+yarn add @commercetools-frontend/application-shell-connectors
+
+# or
+
+npm --save install @commercetools-frontend/application-shell-connectors
+```
+
+Additionally install the peer dependencies (if not present)
+
+```
+yarn add react @apollo/client
+
+# or
+
+npm --save install react @apollo/client
+```
+
+# Context
+
+In this section we describe the type of information you will find in the main application context and how to access it.
```

Comment:
I think it's general wording and that should be consistent (use of `we` for instance).

```suggestion
This section describes different parts of information provided through the application context and how it can be accessed.
```